pkg/Cpanel/Security/Advisor/Assessors/Apache.pm

```diff
@@ -229,6 +234,21 @@ sub _centos_symlink_protection {
             }
         );
     }
+    if ($bluehost_ea4) {
+        $security_advisor_obj->add_advice(
+            {
+                'key'        => 'Apache_bluehost_provided_symlink_protection',
+                'type'       => $warn,
+                'text'       => $self->_lh->maketext('Apache Symlink Protection: the Bluehost provided Apache patch is in effect'),
+                'suggestion' => $self->_lh->maketext(
+                    "It appears that the Bluehost provided Apache patch is being used to provide symlink protection. This is less than optimal. Please review [output,url,_1,Symlink Race Condition Protection,_2,_3].",
+                    'https://go.cpanel.net/EA4Symlink',
```

Comment:
This link doesn't point to a valid document.  If we accept this change and it ends up in 60 for any reason, the Release Team will get an email for a bad go link until the documentation is published.  Please consider publishing the document.